.github/workflows/ci.yml

```
@@ -31,13 +31,13 @@ jobs:
         run: sh ./scripts/danger_lint.sh
   iOS:
     name: iOS ${{ matrix.os }} ${{ matrix.device_name }}
-    runs-on: macos-11
+    runs-on: macos-latest
     needs: [lint]
     strategy:
       matrix:
-        device_name: ["iPhone 12 Pro", "iPad Pro (11-inch) (2nd generation)"]
-        os: ["15.0"]
-        xcode_version: ["13.0"]
+        device_name: ["iPhone 13 Pro", "iPad Pro (11-inch) (3rd generation)"]
+        os: ["16.2"]
+        xcode_version: ["14.2.0"]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
```

If you update to v3 you'll get rid of the warnings in the Annotations section of the Summary: https://github.com/spothero/UtilityBelt-iOS/actions/runs/4571378131

```suggestion
        uses: actions/checkout@v3.5.0
```

---

There are a few of these, not just one that will need to be updated.